### PR TITLE
Dev-8516 homepage 'Getting Started' Section Layout

### PR DESCRIPTION
**Pull request by afnazu**
**High level description:**

Implement Getting Started section layout as part of homepage update

**JIRA Ticket:**
[DEV-8516](https://federal-spending-transparency.atlassian.net/browse/DEV-8516)

**Mockup:**
See ticket for link

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Edge
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [x] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)

Reviewer(s):
- [ ] Code review complete

### CODE REVIEW

**Review by anjenkin — CHANGES_REQUESTED** [review]
Just a couple comments.  Also I think we could add a unit test to check for the existence of content, though doesn't need to be done here but with the corresponding content ticket.

**Comment by anjenkin** [comment]
@afnazu One more thing.  I just double checked with the designers, and we should switch to the mobile view at < 992.  This layout is currently switch to mobile at < 768.

File: src/js/components/homepageUpdate/gettingStarted/GettingStarted.jsx [hunk 1]
```diff
@@ -3,19 +3,109 @@
...
+                    <h3 className="section-heading-title">Getting Started on USAspending</h3>
+                </div>
+                <FlexGridCol className="homepage-spending-explorer" width={12} tablet={6}>
+                    <div className="homepage-spending-explorer__img-wrapper">
+                        Spending explorer feature image
+                    </div>
+                </FlexGridCol>
+                <FlexGridCol className="homepage-spending-explorer" width={12} tablet={6}>
+                    <div className="homepage-spending-explorer-desc">
+                        <h5>A big-picture view of government spending</h5>
```
[the remark this answers is not in the document]
Comment (afnazu) [reply src/js/components/homepageUpdate/gettingStarted/GettingStarted.jsx]: Updated to use <h2> and <h3> organization! I had this updated in the follow up content branch but just updated here for consistency

File: src/js/components/homepageUpdate/gettingStarted/GettingStarted.jsx [hunk 2]
```diff
@@ -3,19 +3,109 @@
...
+                <div className="homepage-search-feature__background-flair" />
+                <div className="homepage-search-feature-desc">
+                    <h5>A targeted approach to finding federal award data</h5>
+                    <p>
+                        Whether you&apos;re a congressional staffer, government employee, researcher, or data buff, our Award Search will help you answer your toughest questions about federal spending.
+                    </p>
+                    <p>
+                        Use Keyword Search for a broad view of award data, but if you want to dig deeper, our Advanced Search offers filters that let you customize your data sets. Interactive visualizations — including a spending map — complement downloadable files.
+                    </p>
+                    <button>Select Search Type</button>
```
[the remark this answers is not in the document]
Comment (afnazu) [reply src/js/components/homepageUpdate/gettingStarted/GettingStarted.jsx:42]: I opened a PR for DEV-8550 which addresses this, once that is merged the existing <button> element will use the updated common style

**Review by anjenkin — APPROVED** [review]

**Pull request merged**